### DISCUSSION TITLE
Do not connect to the server in electron.js

### DISCUSSION THREAD

**Discussion by majidkazemi2000** [socketio/socket.io#3957] [Q&A] [answered]
my code in electron : 
    <script src="https://cdn.socket.io/3.1.3/socket.io.min.js" integrity="sha384cPwlPLvBTa3sKAgddT6krw0cJat7egBga3DJepJyrLl4Q9/5WLra3rrnMcyTyOnh" crossorigin="anonymous"></script>

<script>
    const socket = io('http://localhost');
    console.log(socket);
</script>

////errors in console
/// Uncaught ReferenceError: io is not defined
//// Refused to load the script 'https://cdn.socket.io/3.1.3/socket.io.min.js' because it violates the following Content Security Policy directive: "script-src 'self'". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.

**Comment by darrachequesne** [comment, marked as answer]
Hi! Did you try with:

```js
const { io } = require("socket.io-client");
const socket = io("http://localhost");
```

**Reply by majidkazemi2000** [reply]
tank you . its ok
